### ISSUE TITLE
Consider removing pointermove (and interactionType and pointerIsDragSet) 

### ISSUE THREAD

**Issue by mmocny** [w3c/event-timing#140]
...as used by the [steps of computing interactionID](https://www.w3.org/TR/event-timing/#sec-computing-interactionid)

As far as I can tell, this is computed but isn't actually exposed at all.  It seems not used in computing any other values.

In addition, in Chromium we found that the approach specified can be misleading.  For example, there is a certain movement threshold distance before a pointer interaction actually becomes a "drag" and not a "tap".  The mere dispatch of a move event is not enough signal to make this distinction.  Rather than making this more complicated to account for this, it seems best to just remove this distinction.

(There is is also generally a thin line sometimes between what a user considers a tap or drag, better to just not try to label)

**Comment by canova** [comment]
I was also looking at the `interactionId` computation and was confused by `interactionType`. It seems like we don't do anything with it. It might be good to remove it to remove any confusions.

**Comment by mmocny** [comment]
Yes-- In chromium internally we had tried to use this just for attribution, but in the end the only valuable split we found was keyboard vs pointer events.  This is already available by just looking at the event type names.
